_unittest/test_26_emit.py

```diff
@@ -1,11 +1,16 @@
 # Import required modules
 import os
+import sys
+
+# get aedt delcross python from custom path
+delcross_python_path = os.environ.get("ANSYS_DELCROSS_PYTHON_PATH")
```

Comment:
@apollo-crum Are we adding this to the sys.path so that the `from pyaedt import Emit` line works? If so, can we move this into that module?

Reply:
Sounds good, I'll do that.  I also wasn't sure if there was a better way to do this in general.  I looked for something in common code that can redirect you to a local aedt install but couldn't find anything.